packages/sentry/src/sentry.ts

```diff
@@ -13,15 +30,22 @@ export const boot = () => {
       // from our code and ignore errors that come from other services
       // https://blog.sentry.io/2017/03/27/tips-for-reducing-javascript-error-noise.html
       whitelistUrls: [window.app.cdnUrl, window.app.frontendHost],
+      // we don't need default Sentry's global handlers, because we  add default ones
+      integrations: [
+        new Sentry.Integrations.GlobalHandlers({
+          onunhandledrejection: false,
+          onerror: false,
+        }),
+      ],
     });
-    Sentry.configureScope((scope) => {
+    Sentry.configureScope(scope => {
```

Comment:
Forgot to apply prettier?

Reply:
That's weird, but looks like some outdated prettier/config was applied. Removing `node_modules` and reinstalling all modules helped. 🤷‍♂